### PR TITLE
Parsingrule Update 4

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ApacheWebServer/README.md [hunk 1]
```diff
@@ -1,8 +1,65 @@
...
+
+<~XSIAM>
+## Configuration on Server Side
+You need to configure Apache Web Server to forward Syslog messages.
+
+Open your Apache Web Server instance, and follow these instructions [Documentation](https://docs.trellix.com/bundle/xdr_dscg/page/UUID-4540547f-28c4-0553-846e-544fbc02530f.html):
+1. Log in to your Apache Web Server instance as a **root** user.
+2. Edit the Apache configuration file **httpd.conf**.
+   * Ensure to keep a backup copy of the file.
+   * Edit one config at a time- Save each change at a time and monitor its effect.
```
Comment (ShirleyDenkberg) [inline Packs/ApacheWebServer/README.md:12]:
```suggestion
   * Edit one configuration file at a time. Save the file after each change and monitor its effect.
```

File: Packs/ApacheWebServer/README.md [hunk 2]
```diff
@@ -1,8 +1,65 @@
...
+6. Edit the syslog configuration file:
+```bash 
+   /etc/syslog.conf
+```    
+7. Add the following information to your syslog configuration file:
+```bash 
+   <facility>.<priority> <TAB><TAB>@<host>:<port>
+```
+   Where:
+   * **facility** - This value must match the value that you typed in the previous step.
```
Comment (ShirleyDenkberg) [inline Packs/ApacheWebServer/README.md:35]:
```suggestion
   * **facility** - This value must match the value that you typed in the step 3.
```

File: Packs/ApacheWebServer/ReleaseNotes/1_0_8.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### ApacheWebServer Parsing Rule
+
+- Added a filter for the Parsing Rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/ApacheWebServer/ReleaseNotes/1_0_8.md:6]:
```suggestion
Added a filter for the Parsing Rule to enhance its logic.
```

File: Packs/FireEyeNX/ReleaseNotes/1_1_25.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### FireEyeNX Parsing Rule
+
+- Added a filter for the Parsing Rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeNX/ReleaseNotes/1_1_25.md:6]:
```suggestion
Added a filter for the Parsing Rule to enhance its logic.
```

File: Packs/IvantiConnectSecure/ReleaseNotes/1_0_1.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+#### Parsing Rules
+
+##### Ivanti Connect Secure Parsing Rules
+
+- Added a filter for the Parsing Rule to enhance its logic.
```
Comment (ShirleyDenkberg) [inline Packs/IvantiConnectSecure/ReleaseNotes/1_0_1.md:6]:
```suggestion
Added a filter for the Parsing Rule to enhance its logic.
```

**Comment by ShirleyDenkberg** [comment]
@evisochek Doc review completed.

**Review by evisochek — APPROVED** [review]

**Pull request merged**